### ISSUE TITLE
[Unstable] Getting overvolting benefits is practically impossible + OV feedback

### ISSUE THREAD

**Issue by ubertpendragon** [Regalis11/Barotrauma#9732]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

`- Overvoltage makes devices perform better, increasing the output of engines, making fabricators, deconstructors and fabricators operate faster, electrical discharge coils do more damage and oxygen generators generate more oxygen. Incentivizes operating the reactor manually and hopefully makes it a little more engaging.`

So, a few observations first:

-Overvolting without overheating/burning is dependent on the amount of rods used; almost impossible with only 1/2 Uranium rods 
-Oxygen generation is a non-factor for reasons inherent to how O2 generation/consumption is handled. More on that later
-Discharge coil seem to deal 25% extra damage? I couldn't really tell, it probably needs to be much higher than that. 
-Fabrication speed seems to go up 100%. Hard to say if it's worth burning electrical but at least it's good enough.
-Engine speed is noticeable and always welcome, although riskier than Tinkering which is a T1 talent. However...

### It is practically impossible to benefit from Overvolting bonuses with the current reactor system.

Using default values and on most vanilla submarines the engine power consumption represents 30 to 40% of the sub's max reactor output, with O2 representing around 10 to 15%

To overvolt the submarine, you roughly need **Power = 2xLoad**

To put it simply, your engine creates so much load, attaining a state of 2x the load would require you to cut off all other electrical systems. 
With the current reactor system and on a "balanced" submarine, max output will often match the load of a submarine going at full speed. **The only way to reliably overvolt is to cut the engine and/or O2 generator, both of which are beneficiaries...**

It mostly has to do with how reactors are limited within the confines of max output as talked about in #8702. You simply can't freely produce enough power to reliably overvolt when the sum of machines running is >50% of your max output

**Comment by Regalis11** [comment]
Thank you for the feedback!

I don't quite agree with the overvolting being impossible to benefit from. You do not need to reach x2 load - that is the maximum overvoltage you can reach before things start breaking, so in practice you should try to stay below that.

**Comment by ubertpendragon** [comment]
> You do not need to reach x2 load - that is the maximum overvoltage you can reach before things start breaking, so in practice you should try to stay below that. 

Do you mean you get the benefits from simply having Load > Power, and not when the junction boxes turn red?

**Comment by Regalis11** [comment]
The other way around: overvoltage occurs when the power output is higher than the load. But basically yeah, you don't need to go all the way up to x2. Any amount of overvoltage improves the performance of the devices, and it increases linearly as the voltage increases (up until you get to the point where they start taking damage).

**Comment by ubertpendragon** [comment]
I assumed the overvolt effect only occured after reaching the 2x threshold, which sounded more exciting.

Will have to see if permanent overvolting actually rewards ARC automation more than manual control on the long run
